connectors/byoc.py

```diff
@@ -528,7 +528,7 @@ async def _sync_done(self, job, result, exception=None):
         self.doc_source["last_synced"] = iso_utc()
         await self.sync_doc()
         logger.info(
-            f"Sync done: {indexed_count} indexed, {doc_deleted} "
+            f"[{self.id}] Sync done: {indexed_count} indexed, {doc_deleted} "
```

Comment:
We'll need to do it in a lot of places with this change, PR to track: https://github.com/elastic/enterprise-search-team/issues/3809